### ISSUE TITLE
syscall not implemented warnings

### ISSUE THREAD

**Issue by stfairy** [lkl/linux#21]
I checked out 2b6ea3c and did

```sh
cd tools/lkl
make
```

and I'm seeing warnings like this

```
  CALL    scripts/checksyscalls.sh
<stdin>:257:2: warning: #warning syscall setuid not implemented [-Wcpp]
<stdin>:311:2: warning: #warning syscall dup not implemented [-Wcpp]
<stdin>:317:2: warning: #warning syscall times not implemented [-Wcpp]
<stdin>:326:2: warning: #warning syscall setgid not implemented [-Wcpp]
<stdin>:335:2: warning: #warning syscall geteuid not implemented [-Wcpp]
<stdin>:338:2: warning: #warning syscall getegid not implemented [-Wcpp]
<stdin>:341:2: warning: #warning syscall acct not implemented [-Wcpp]
<stdin>:353:2: warning: #warning syscall fcntl not implemented [-Wcpp]
<stdin>:359:2: warning: #warning syscall setpgid not implemented [-Wcpp]
<stdin>:380:2: warning: #warning syscall getppid not implemented [-Wcpp]
<stdin>:386:2: warning: #warning syscall setsid not implemented [-Wcpp]
<stdin>:398:2: warning: #warning syscall setreuid not implemented [-Wcpp]
<stdin>:401:2: warning: #warning syscall setregid not implemented [-Wcpp]
<stdin>:410:2: warning: #warning syscall sethostname not implemented [-Wcpp]
<stdin>:413:2: warning: #warning syscall setrlimit not implemented [-Wcpp]
<stdin>:416:2: warning: #warning syscall getrlimit not implemented [-Wcpp]
<stdin>:419:2: warning: #warning syscall getrusage not implemented [-Wcpp]
<stdin>:422:2: warning: #warning syscall gettimeofday not implemented [-Wcpp]
<stdin>:425:2: warning: #warning syscall settimeofday not implemented [-Wcpp]
<stdin>:428:2: warning: #warning syscall getgroups not implemented [-Wcpp]
<stdin>:431:2: warning: #warning syscall setgroups not implemented [-Wcpp]
<stdin>:467:2: warning: #warning syscall ftruncate not implemented [-Wcpp]
```
...

Are they really not implemented yet, or the warnings are just spurious?

**Comment by thehajime** [comment]
as far as I know, those calls are planed (tm) to implement in lkl at some point.
indeed, it's annoying and makes other errors hard to catch though.

**Comment by tavip** [comment]
Fixed in commit 5c865ecdeeaf58c18d01afb75eb36f4f4f7bf2da
